docs/elasticsearch-apis.md

```diff
@@ -0,0 +1,122 @@
+# Elasticsearch APIs
+
+1. **Ingest API**:
+Ingest data in JSON format. Supports reading json data from file.
+
+    Example:
+
+    Using json file:
+    ```
+    curl -X POST http://localhost:8081/elastic/_bulk \
+    -H 'Content-Type: application/json' \
+    --data-binary "@sampleEvents.json" 
+    ```
+    Using raw json:
+    ```
+    curl -X POST "http://localhost:8081/elastic/_bulk" \
+    -H 'Content-Type: application/json' \
+    -d '{ "index" : { "_index" : "test" } }
+    { "name" : "john", "age":"23" }'
+    ```
+
+
+2. **Get elasticsearch cluster information**:
+
+    ```
+    curl  http://localhost:8081/elastic/
+    curl -X GET "http://localhost:8081/elastic/"
+    curl -X GET "http://localhost:8081/elastic/_xpack"
```

Comment:
do we suppor txpack ?

Reply:
yes, the `elastic/_xpack` endpoint is defined in server.go. Returns same output as `/elastic/` endpoint